game.gd

```diff
@@ -9,8 +9,16 @@ func _ready():
 	network.update_maps()
 	
 	screenfx.play("fadein")
+	
+	# defer these connections to make sure all elements are attached and ready
+	call_deferred("_connect_transitions")
+
+func _connect_transitions():
+	# On these 2 signals, we want to check whether we want to activate enemies or not
+	screenfx.connect("animation_finished", self, "_set_enemy_physics_processes")
+	$Camera.connect("screen_change_completed", self, "_set_enemy_physics_processes")
```

Comment:
Probably want to verify that this $Camera node is present.
Maybe call the enemy_physics process regardless after x ticks, to be safe, in case some wonky camera stuff happens later